state_management/go/http/order-processor/app.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"math/rand"
+	"net/http"
+	"os"
+	"strconv"
+	"time"
+)
+
+func main() {
+	DAPR_HOST := "http://localhost"
+	_, okHost := os.LookupEnv("DAPR_HOST")
```

Comment:
Could simplify sightly to:

`
if DAPR_HOST, ok = os.LookupEnv("DAPR_HOST"); !ok {
    DAPR_HOST = "http://localhost"
}
`